src/components/UserInfo/UserInfo.jsx

```diff
@@ -1 +1,9 @@
-export const UserInfo = () => <>Put the user here</>;
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => (
+  <>
```

Comment:
same

Reply:
Can you explain to me, what is a redundant fragment? If you mean that - <></>, so I removed these fragments last time, or I don't understand what do I need to do.